src/sidebar/pin/command/UnpinAllFromSidebarCommand.ts

```diff
@@ -0,0 +1,13 @@
+import Command from "../../../common/vscode/command/Command";
+import { askUnpinAll } from "../flow/pinFlow";
+
+class UnpinAllFromSidebarCommand implements Command {
+    public name = "zeplin.sidebar.unpinAll";
+
+    public execute() {
+        askUnpinAll();
+        return Promise.resolve();
+    }
```

Comment:
Protip: You can make a function async to return promise.
```suggestion
    public async execute() {
        askUnpinAll();
    }
```

Reply:
lint blocks that

Reply:
Shame..